### DISCUSSION TITLE
Any example of Flask-SocketIO that include a simple Login page or Flask-Login?

### DISCUSSION THREAD

**Discussion by lonnietc** [miguelgrinberg/Flask-SocketIO#1605] [Q&A]
Hello,

I am investigating Flask-SocketIO for use in a very small project.

The project has a server that receives POST data as a webhook from another site and then dynamically updates the client displaying rows of data as the come in from the external site.

I can do this without dynamically updating the client browser (i.e. no server push) which is why I am looking at Flask-SocketIO that seems to do a great job of server pushes to the client, but I am not clear on how to setup a simple login page. It could have even a static login/password since only one person will be using it thus now multi-user login support is needed.

Are there any simple examples of Flask-SocketIO with a Login page?
Thanks in advance

**Comment by miguelgrinberg** [comment]
Nothing prevents you from implementing the login over Socket.IO, but in general applications perform the authentication through a standard login form, and only then make the Socket.IO connection, which is authenticated either by a cookie set when the login form was accepted, or by a token returned to the client.

**Comment by lonnietc** [comment]
Thanks for responding so quickly to this.

Being still new to Python and also Flask-SocketIO, I will will to investigate this a bit further to see what can be done.  Since only one person, me, will be using this, I just wanted to try and figure out some reasonable simple security to prevent unauthorized people from accessing the site.

I will dig into it more to see what I can figure out since I am not sure yet how to do it and was hoping to find some simple example somewhere.

Thanks again  and have a great day/night.
